features/promotion/managing_coupons/coupon_unique_code_validation.feature

```diff
@@ -0,0 +1,21 @@
+@managing_promotion_coupons
+Feature: Coupon unique code validation
+    In order to uniquely identify coupons
+    As an Administrator
+    I want to be prevented from adding two coupons with the same code
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has promotion "Christmas sale" with coupon "Santa's gift"
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Trying to add coupon with taken code
+        Given I want to create a new coupon for this promotion
+        When I specify its code as "Santa's gift"
+        And I set its usage limit to 30
+        And I set its per customer usage limit to 25
```

Comment:
And I limit its usage to 25 times per customer
And I limit its overall usage to 30 times

WDYT?